### DISCUSSION TITLE
help for creating custom project template

### DISCUSSION THREAD

**Discussion by neozhu** [jasontaylordev/CleanArchitecture#888] [Show and tell]
Dear all,

I am reaching out to kindly request your assistance in creating a template for my project. Regrettably, I lack knowledge in this specific area and despite several attempts, I have been unsuccessful.

For reference, the structure of my project closely mirrors that of this project. As such, I would greatly appreciate it if you could assist me in setting up the template within my own project. https://github.com/neozhu/CleanArchitectureWithBlazorServer

Thank you so much in advance for your time and support. I am looking forward to your insights and advice.

Best regards,
Hualin

**Comment by jasontaylordev** [comment]
These are the best resources I have found:

- https://learn.microsoft.com/en-us/dotnet/core/tools/custom-templates
- https://github.com/dotnet/templates
- https://github.com/dotnet/templating
- https://github.com/sayedihashimi/template-sample
